cmd/config-shard-validator/main.go

```diff
@@ -123,6 +124,15 @@ func main() {
 		var matchedMap string
 		logger := logrus.WithField("source-file", pathToCheck.path)
 		for glob, updateConfig := range pcfg.ConfigUpdater.Maps {
+			if _, hasDefaultCluster := updateConfig.Clusters[prowv1.DefaultClusterAlias]; hasDefaultCluster {
+				logger.Errorf("cluster alias %s is not allowed, please explicitly specific the name of the cluster", prowv1.DefaultClusterAlias)
+				foundFailures = true
+			}
+			if updateConfig.GZIP == nil || !*updateConfig.GZIP {
```

Comment:
currently we don't zip everything, only jobs -- need to either make this check conditional or fix the config first